icicle/backend/cpu/src/curve/cpu_msm.hpp

```diff
@@ -352,20 +421,23 @@ void Msm<A, P>::phase1_bucket_accumulator(const scalar_t* scalars, const A* base
       // Handle required preprocess of base P
       A base =
         m_are_points_mont ? A::from_montgomery(bases[m_precompute_factor * i + j]) : bases[m_precompute_factor * i + j];
+      // TODO move to preprocess before precompute to avoid repeating conversions
```

Comment:
todo when? also for the one below

Reply:
Maybe not this week, I forgot to remove TODOs before the PR.